debian/changelog

```diff
@@ -1,8 +1,116 @@
-glib2.0 (2.46.2-0) master; urgency=medium
+glib2.0 (2.48.0-0) master; urgency=medium
```

Comment:
I found that there's actually no need to do this - as long as the tags are there, Jenkins will happily override the version number without the need to tweak the changelog.